codacy/values.yaml

```diff
@@ -51,6 +51,11 @@ global:
     # protocol: "https"
     # port: 443
 
+  bitbucket: {}
+    # enabled: "false"
+    # key: ""
+    # secret: ""
+
```

Comment:
This is also done for the other Git providers, but do we need to have these settings repeated on both the `values.yaml` and `values-production.yaml` files?

On the [installation instructions](https://codacy.github.io/chart/#2-installing-codacy) we're asking the customers to configure their settings on the `values-production.yaml` file. So I think it's less confusing to have the settings only in one place.

Reply:
hmm then I don't know the purpose of having these values.yaml if we always use values-production.yaml. Is the values.yaml just for reference? @h314to @paulopontesm @joaocosta-codacy

Reply:
The `values.yaml` file is a default configuration file, that can be overwritten by other `.yaml` files that are passed explicitly as parameters to the `helm` command:

https://helm.sh/docs/chart_template_guide/values_files/

So the logic being followed so far is that we use the `values.yaml` the set the defaults but all the configurations that must be configured by the user are specified in the `values-production.yaml` (and `values-microk8s.yaml` as well if the user is installing on MicroK8s).

But let's wait for someone from the Reliability team to confirm this.